File: Examples/test-suite/ruby/rev_len_str_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#

require 'swig_assert'
require 'rev_len_str'

include Rev_len_str

t = Test.new
str = 'hile'
swig_assert(t.strlen(str) == 4, binding, 'bad multi-arg typemap')
swig_assert(t.strlen("hil\000") == 4, binding, 'bad multi-arg typemap')

File: Examples/test-suite/octave/rev_len_str_runme.m
rev_len_str

t = rev_len_str.Test();
str = "hile";
if (t.strlen(str) != 4)
  disp(t.strlen(str));
  error("bad multi-arg typemap");
endif

if (t.strlen("hil\000") != 4)
  error("bad multi-arg typemap");
endif
